frontend/packages/ceph-storage-plugin/src/components/dashboards/odf-system-dashboard.tsx

```diff
@@ -29,7 +29,8 @@ const ODFSystemDashboard: React.FC<ODFSystemDashboardPageProps> = ({
   const isObjectServiceAvailable = useFlag(MCG_FLAG);
   const isCephAvailable = useFlag(CEPH_FLAG);
   const { systemName } = rest.match.params;
-  const dashboardTab = isCephAvailable === false && isObjectServiceAvailable ? OBJECT : BLOCK_FILE;
+  const dashboardTab =
+    !!isCephAvailable === false && isObjectServiceAvailable ? OBJECT : BLOCK_FILE;
```

Comment:
I think the fix your BZ would be suffice by this ?
`undefined` is a falsy value.

```suggestion
    !isCephAvailable && isObjectServiceAvailable ? OBJECT : BLOCK_FILE;
```